src/test/java/com/google/api/codegen/testdata/python_doc_message_library.baseline

```diff
@@ -1293,8 +1293,8 @@ class Comment(object):
     Attributes:
       user_name (string): won't be filled in by the sample generator
       comment (bytes): should be filled in by the sample generator
-      stage (enum :class:`google.cloud.gapic.example.library.v1.enums.Comment.Stage`): should be filled in by the sample generator
-      alignment (enum :class:`google.cloud.gapic.example.library.v1.enums.SomeMessage2.SomeMessage3.Alignment`): Tests Python nested enums
+      stage (enum :class:`google.cloud.example.library_v1.gapic.enums.Comment.Stage`): should be filled in by the sample generator
```

Comment:
Let's change all of these to use the `~.` form. In a perfect world, drop everything before `enums`; if that is too hard, drop everything before the final period.

Best: `~.enums.Comment.Stage`
Good enough: `~.Stage`